install.sh

```diff
@@ -218,16 +218,29 @@ do_install_rpm() {
     if [ "${1}" = "testing" ]; then
         rpm_site="rpm-${1}.rancher.io"
     fi
+    maj_ver="7"
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
+        dist_version="$(. /etc/os-release && echo "$VERSION_ID")"
+        maj_ver=$(echo "$dist_version" | sed -E -e "s/^([0-9]+)\.?[0-9]*$/\1/")
+        case ${maj_ver} in
+            7|8)
+                :
+                ;;
+            *) # In certain cases, like installing on Fedora, maj_ver will end up being something that is not 7 or 8
+                maj_ver="7"
```

Comment:
In case of Fedora or other bleeding-edge distros, shouldn't we default to the EL8 packages?

Reply:
There is a case to be made to rely on the EL8 packages for fedora, but due to the fact that you must use `2.124` or newer of `container-selinux` with the `el8` `rke2-selinux` compiled policy, I didn't want to force the hand in the event that someone was running an older Fedora version. The `el7` policies/rpms are identical to the `el8` rpms and should work on all fedora systems.

Reply:
As long as they work!